### DISCUSSION TITLE
Importing songs

### DISCUSSION THREAD

**Discussion by LOUDKaan** [fast4x/RiMusic#3231] [Q&A]
Hi, I recently updated the app RiMusic, and I exported all my playlists, how can I import them back, thanks.

**Comment by fast4x** [comment]
look
![Screenshot_2024-08-13-13-45-29-271_it.fast4x.rimusic-edit.jpg](https://github.com/user-attachments/assets/354907ef-a74e-48d6-812e-7e68517e770a)
